runtime/runtime/src/state_viewer/mod.rs

```diff
@@ -22,14 +22,21 @@ use crate::{actions::execute_function_call, ext::RuntimeExt};
 
 pub mod errors;
 
-pub struct TrieViewer {}
+pub struct TrieViewer {
+    /// Upper bound of the byte size of contract state that is still viewable. None is no limit
+    state_size_limit: Option<u64>,
```

Comment:
Unsure if we need an `Option` here. Practically speaking, no limit and very large limit are indistinguishable, so maybe it makes sense to just use `u64` here and let the caller pass `u64::MAX`?

Reply:
u64::MAX is not easy to pass in config.json. while this one it's "trie_viewer_state_size_limit": null

Reply:
I guess, we can have it as `Option` in the config.json, and then to `limit.unwrap_or(u64::MAX)` when converting from config to something we pass around. But yeah, it's also convenient to specify `None` as a default for tests. I guess I don't have a strong opinion one way or another here.

Reply:
I feel to say" limit is None" is more intuitive than limit is "u64::max" so i just passed that around